R/auditClassicalEstimation.R

```diff
@@ -128,23 +128,27 @@ auditClassicalEstimation <- function(jaspResults, dataset, options, ...) {
   regressionTable$addColumnInfo(
     name = "estimate",
     title = gettextf("Estimate %1$s", "W&#770"),
-    type = "number"
+    type = "number",
+    format = "monetary"
```

Comment:
Dont you want to tell it the currency?

Reply:
I don't understand, it is euro by default right?

Reply:
Yes but I thought I remembered seeing a "currency selector" in one of your analyses?

Reply:
But maybe you removed it, in that case this is fine yeah

Reply:
No this removed was a very long time ago ;)